db/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+inherit_from:
```

Comment:
hmm I'm not really convinced about having multiples .rubocop.yml overriding each other.. I feel like just a centralized place is simpler.. not sure though

Reply:
I'm not sure that this file is in `db/`, I would move it to the main folder

Reply:
Agree, we should move it to the root.

We can customize the root `.rubocop.yml` file using [Include/Exclude](https://docs.rubocop.org/rubocop/1.55/configuration.html#cop-specific-include-and-exclude) to enable some cops only for specific directories